client/lib/desktop/index.js

```diff
@@ -86,20 +86,26 @@ var Desktop = {
 		ipc.send( 'unread-notices-count', count );
 	},
 
-	clearNotificationBar: function() {
-		// todo find a better way. seems to be react component state based
-		var bar = document.querySelector( '#wpnt-notes-panel2' );
+	getNotificationLinkElement: function() {
+		return document.querySelector( '.masterbar__item-notifications' );
+	},
 
-		if ( bar.style.display === 'block' ) {
-			this.onToggleNotifications();
+	clearNotificationBar: function() {
+		// TODO: find a better way. seems to be react component state based
+		let notificationsLink = this.getNotificationLinkElement();
```

Comment:
Super minor, but when we have an identifier that we don't reassign to (`notificationsLink`, in this case), we generally use `const`.